tests/Fixtures/Integration/priority/binary_operator_spaces,union_types_spaces.test

```diff
@@ -0,0 +1,17 @@
+--TEST--
+Integration of fixers: binary_operator_spaces,union_types_spaces.
+--RULESET--
+{"binary_operator_spaces": true, "union_types_spaces": true}
```

Comment:
IMO we shouldn't have this priority constraint: `binary_operator_spaces` should ignore union types. I don't even thinks `|` can be considered an operator when used in union types.

Reply:
Could be, yet there are these PRs: #3210 and #3215 - should be consider them wrong?

Reply:
I would.